parachain/primitives/core/src/types.rs

```diff
@@ -26,8 +26,10 @@ pub enum ChannelId {
 pub struct Message {
 	/// The raw RLP-encoded message data.
 	pub data: Vec<u8>,
-	/// Input to the message verifier
+	/// Input to the message verifier.
 	pub proof: Proof,
+	/// Untrusted weight declared by relayer.
+	pub weight: Weight,
```

Comment:
Let's rename this field to `dispatch_weight` in this struct, so that readers know its referring to the weight of the dispatchable call.

Also, lets expand on the comment a bit: The declared weight of the dispatchable call, extracted from the message payload by relayers. This is untrusted, so the message payload will be rejected if the actual dispatchable weight is higher than the declared weight.

Reply:
Renamed in 8ccbef4c & comment updated in c9afd97e